prep_local

```diff
@@ -10,7 +10,7 @@ set -o errexit -o nounset -o pipefail
 : ${VIRTUAL_ENV?"Must be run in a python virtual environment"}
 
 # Install the latest version of pip
-pip install pip==8.1.2
+pip install -U pip
```

Comment:
We ran into a issue with doing `pip install pip==8.1.2` when pip 8.1.1 itself was broken, when it could not parse a metadata changed higher version of pip in 8.1.2.  The `-U` (or complete reinstall) was the reasonable way.

In general, I agree with pinning the version, but for the pip's upgrade case. `-U` seems alright to me as that's what is suggested by pip docs.